project/graph_utils.py

```diff
@@ -0,0 +1,24 @@
+import cfpq_data
+from collections import namedtuple
+from typing import Tuple
+import networkx
+
+GraphInfo = namedtuple("GraphInfo", ["vertices", "edges", "labels"])
+
+
+def get_graph_info(graph) -> GraphInfo:
```

Comment:
Если используете `type hints`, то старайтесь делать это везде.
В основном вы будете работать с `nx.MultiDiGraph`. Но можете также прописать `Union` всех типов графов из `networkx`.
```suggestion
def get_graph_info(graph: nx.MultiDiGraph) -> GraphInfo:
```